org/mozilla/jss/pkcs11/PK11Signature.c

```diff
@@ -90,20 +133,67 @@ Java_org_mozilla_jss_pkcs11_PK11Signature_initVfyContext
 {
 	VFYContext *ctxt=NULL;
 	jobject contextProxy=NULL;
-	SECKEYPublicKey *pubk;
+        SECKEYPublicKey *pubk;
```

Comment:
We should probably keep the same indentation as the rest of the file.

Reply:
I don't understand this one. If you look at the file, it's all aligned..

Reply:
The issue is tabs versus spaces. The rest of this function uses tabs I think, but you've inserted spaces.

Try `:set list` in vim to show tabs/spaces. Not sure about other editors. :)

Reply:
I missed this one. Soon I will have to move on to pki :)